pkg/cephfs/volume.go

```diff
@@ -104,12 +105,22 @@ func createVolume(volOptions *volumeOptions, cr *util.Credentials, volID volumeI
 		strconv.FormatInt(bytesQuota, 10),
 		"--group_name",
 		csiSubvolumeGroup,
-		"--pool_layout", volOptions.Pool,
-		"--mode", "777",
-		"-m", volOptions.Monitors,
-		"-c", util.CephConfigPath,
-		"-n", cephEntityClientPrefix+cr.ID,
-		"--keyfile="+cr.KeyFile)
+	}
+
+	if volOptions.Pool != "" {
```

Comment:
This option can be added to the end of the CLI args, so no need to split the command at exactly this point to append the same conditionally.